src/clib/pio_darray_int.c

```diff
@@ -1521,8 +1521,7 @@ int flush_output_buffer(file_desc_t *file, bool force, PIO_Offset addsize)
 
     /* Find out the buffer usage. If I check the turn code, some tests fail. ??? */
     if ((ierr = ncmpi_inq_buffer_usage(file->fh, &usage)))
-        return ierr;
-    /*return pio_err(ios, file, PIO_EBADID, __FILE__, __LINE__);*/
+	return ierr;
```

Comment:
Was this pio_err commented out because non-io tasks were getting here?  Perhaps it can be put back in.